pages/common/bpython.md

```diff
@@ -0,0 +1,21 @@
+# bpython
+
+> A fancy interface to the Python interpreter.
+> Provides syntax highlighting and many other nice-to-haves in REPL mode.
+> More information: <https://docs.bpython-interpreter.org/en/latest/>.
```

Comment:
Is there a better link? This one doesnt show information about how to use the tool

Reply:
There's also <https://bpython-interpreter.org/> but I don't know if it is more helpful.

Reply:
Nevermind, I found https://manned.org/man/bpython

Should we take this one even though it is not "official"?

Reply:
I would say we should use this https://docs.bpython-interpreter.org/en/latest/ as it is official and will be actively maintained.